### ISSUE TITLE
WCAG 2.2 Error with class .hmrc-sign-out-nav__link

### ISSUE THREAD

**Issue by thirlz95** [hmrc/hmrc-frontend#337]
I'm using the header component within my project and I've come across the following serious warning from accessibility testing with the new WCAG 2.2 guidelines. 

![Screenshot 2024-01-10 at 13 59 02](https://github.com/hmrc/hmrc-frontend/assets/20689243/d2fec63c-f8b5-4ac6-bff8-152fb5aa6bb4)

I believe this could be fixed by increasing the font-size from 1rem to 1.2rem. I'd happily create a PR for this if you wish.

**Comment by YashuMishra** [comment]
Yeah it can be fixed but you can do one more thing increase the padding around the target element to effectively increase its clickable area.

**Comment by ellamdav** [comment]
Thanks for reporting this, and apologies for the delay in getting to it 🙌

Are you able to confirm that this is still an issue with the latest version?  If so, could you possibly include a screenshot of the problem page and/or a link to your repo, or just the relevant HTML fragment?

It looks like maybe your Service Name is a clickable link and is coming up against the sign-out link, but I've been unable to reproduce this locally.

Can you also confirm which accessibility tool(s) are showing the violation?

**Comment by thirlz95** [comment]
I can confirm that the latest update has resolved the issue, closing now
